compiler-rt/test/radsan/TestCases/test_radsan.cpp

```diff
@@ -5,7 +5,7 @@
 #include <stdlib.h>
 
 [[clang::realtime]] void violation() {
-    auto x = malloc(2);
+    void* Ptr = malloc(2);
```

Comment:
did a big find/replace because clang tidy was yelling at me. This is apparently clang style for variable name is UpperCamelCase

Reply:
Gosh, I've never seen that style before!

Reply:
I find it.....horrible lol. I agree. I have seen UpperCamelCase for functions before, but variables are normally snake or lowerCamel

Reply:
Ha yeah - it's not one I would have chosen myself, but hey ho, consistency is more important. I took a peek at some of the clang code that I hadn't looked at in a while and saw that there's at least `auto` allowed - so we could probably get away with `auto * Ptr` without clang tidy complaining too much. But it's not a big deal, don't worry about changing this if you're not already making changes! I'll resolve this conversation now 👍